updatecli/updatecli.d/update-k8s-k3s/update-k3s.yaml

```diff
@@ -0,0 +1,48 @@
+name: '{{ .k8s_k3s.manifest }}'
+pipelineid: update-k8s-k3s-versions
+
+conditions:
+  check-dockerfile-k3s-cattle-version:
+    name: 'Check if ENV CATTLE_K3S_VERSION is set in Dockerfiles'
+    kind: file
```

Comment:
I guess you didn't use the Dockerfile plugin because it's not possible to only check if the env is set while the current plugin also check the value https://www.updatecli.io/docs/plugins/resource/dockerfile/

That's could be an easy small improvement similar to the yaml plugin where we only want to check if a key exist no matter the value

Reply:
Yes, you are right, that's why I had to use the `file` kind. I only wanted to be sure that the ENV var is set in the file and at the condition I don't care with its value.